### ISSUE TITLE
Typing errors when compiling

### ISSUE THREAD

**Issue by tdranv** [relative-ci/bundle-stats#2821]
Hey, any idea why I'm getting the following errors when I compile my project:

```
node_modules/bundle-stats-webpack-plugin/typings.d.ts(2,68): error TS1183: An implementation cannot be declared in ambient contexts.
node_modules/bundle-stats-webpack-plugin/typings.d.ts(4,19): error TS2304: Cannot find name 'Compiler'.
node_modules/bundle-stats-webpack-plugin/typings.d.ts(5,2): error TS1036: Statements are not allowed in ambient contexts.
node_modules/stylelint-webpack-plugin/types/options.d.ts(16,13): error TS2315: Type 'Plugin' is not generic.1
```

I'm using `bundle-stats-webpack-plugin` version 4.1.2 and `react` 18.2.0.

I see that [here](https://github.com/relative-ci/bundle-stats/issues/1893) a method was added but `Compiler` is not imported from anywhere.

Setting `skipLibCheck` to true in the tsconfig fixes it.

**Comment by vio** [comment]
Hey @tdranv, thank you for creating the issue! What version of TS do you use?

**Comment by tdranv** [comment]
Hey, sorry for forgetting to add that. I'm on 4.5.2.

**Comment by tdranv** [comment]
It seems to be fixed now. Thank you so much. :)

**Comment by vio** [comment]
@tdranv thank you for confirming! The fix was included in the [v4.1.4 release](https://github.com/relative-ci/bundle-stats/releases/tag/v4.1.4) ;)
